src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -1111,8 +1103,7 @@ public function testMultipleChoiceExpandedAttributes()
         $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array('&a', '&c'), array(
             'choices' => array('Choice&A' => '&a', 'Choice&B' => '&b', 'Choice&C' => '&c'),
             'choice_attr' => array('Choice&B' => array('class' => 'foo&bar')),
-            'multiple' => true,
-            'expanded' => true,
+            'widget' => 'checkbox',
```

Comment:
Same for div layout. Missing some legacy tests and new tests for `widget` when `text` or `hidden`.